validation_pipeline.ipynb

```diff
@@ -7,7 +7,7 @@
         "colab_type": "text"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.    plt.subplot(1, 2, 2)
</pre><p>still a lot of code duplication here.</p><p> </p><p>whatever is duplicated should be turned into a method</p><p>as in:</p><pre class="ql-syntax" spellcheck="false">plt.subplot(1, 2, 1)

if len(oxygen_means_isoscape.masked_image.shape) &gt; 2:
&nbsp;raster.plot_band(
&nbsp;&nbsp;oxygen_means_isoscape,
&nbsp;&nbsp;0)
else:
&nbsp;raster.plot_band(
&nbsp;&nbsp;oxygen_means_isoscape,
&nbsp;&nbsp;-1)
plt.xlabel("Longitude")
plt.ylabel("Latitude")
plt.title(ISOSCAPE_OXYGEN_MEANS_FILENAME, fontsize=7)


plt.subplot(1, 2, 2)
if len(oxygen_means_isoscape_comparison.masked_image.shape) &gt; 2:
&nbsp;raster.plot_band(
&nbsp;&nbsp;oxygen_means_isoscape_comparison,
&nbsp;&nbsp;0)
else:
&nbsp;raster.plot_band(
&nbsp;&nbsp;oxygen_means_isoscape_comparison,
&nbsp;&nbsp;-1)
plt.title(ISOSCAPE_OXYGEN_FOR_COMPARISON, fontsize=7)

plt.show()
</pre><p> </p><p>should be:</p><p> </p><pre class="ql-syntax" spellcheck="false">def make_subplot(index: int, oxygen_means_isoscape
  plt.subplot(1, 2, index)
  
  if len(oxygen_means_isoscape.masked_image.shape) &gt; 2:
  &nbsp;raster.plot_band(
  &nbsp;&nbsp;oxygen_means_isoscape,
  &nbsp;&nbsp;0)
  else:
  &nbsp;raster.plot_band(
  &nbsp;&nbsp;oxygen_means_isoscape,
   &nbsp;-1)

make_subplot(1, oxygen_means_isoscape)
make_subplot(2, oxygen_means_isoscape_comparison)
plt.xlabel("Longitude")
plt.ylabel("Latitude")
plt.title(ISOSCAPE_OXYGEN_MEANS_FILENAME, fontsize=7)
plt.show()
</pre><p> </p><p>this is not only a smaller amount of code, but it also makes it easier to later make changes to the subplots.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tnc-br/ddf-isoscapes/pull/179/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>